pkgs/applications/radio/rbfeeder/default.nix

```diff
@@ -0,0 +1,71 @@
+{ lib, stdenv
+, fetchFromGitHub
+, pkg-config
+, curl
+, glib
+, jansson
+, ncurses
+, protobuf
+, protobufc
+, hackrf
+, libbladeRF
+, libusb1
+, rtl-sdr
+, limesuite
+, DarwinTools
+}:
+
+stdenv.mkDerivation rec {
+  pname = "rbfeeder";
+  version = "1.0.8";
+
+  src = fetchFromGitHub {
+    owner = "airnavsystems";
+    repo = pname;
+    rev = "17a5f65da1b9ca1a01c636eb728ea6db9010b708";
```

Comment:
Why are you using raw hashes instead of `v${version}`?

Reply:
It doesn't look like that particular repository provides tags.... Is it even possible to do what you are suggesting in such case?

Reply:
If the repo does not provide tagged snapshots, then it is impossible indeed.

It evokes another question: how do you know or infer this `version =  1.0.8` for this software?
Usually when using raw `rev`s, we put `version = unstable-YYYY-MM-DD` indicating the date the code was created.